### ISSUE TITLE
Splitting multiple 'skip' paths onto indented config file lines behaves differently than when on a single line

### ISSUE THREAD

**Issue by oddhack** [codespell-project/codespell#3399]
The INI file format claims to support an option value that is split across multiple lines, if the lines after the first are indented. However, I see different behavior with the two settings

```
skip=*.adoc,./build_tests/*,
```

```
skip=*.adoc,
    ./build_tests/*,
```

The first works (skips a file with misspellings under build_tests/BAD), the second fails (reports the misspelling). Am I missing something about the INI file format that would allow this to work?

Attaching a zipfile with this minimal example.

[codespell-config.zip](https://github.com/codespell-project/codespell/files/14926060/codespell-config.zip)

**Comment by DimitriPapadopoulos** [comment]
As far as I know, INI files lack a unified standard, and we rely on a Python library to read them. I'll have a look though,  it might be just a matter of passing an option to that library.

Meanwhile, where exactly have you read the claim to support an option value that is split across multiple lines?

**Comment by DimitriPapadopoulos** [comment]
In the multi-line case, [configparser.ConfigParser.read](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read) keeps the newline. After adding this debug line:
```python
    print(config["codespell"]["skip"])
```
right after:
https://github.com/codespell-project/codespell/blob/69c0f002e9e43ed92f31fa19e41bde5a96b065a2/codespell_lib/_codespell.py#L649
I see a comma-separated list of directories to skip in the single-line case:
```
*.adoc,./build_tests/*,
```
while in the multi-line case, the newline is kept in addition to the commas:
```
*.adoc,
./build_tests
```

We need to discard the newlien at some point.

**Comment by oddhack** [comment]
The documentation is a little vague on whitespace handling in INI files. This seems to be a case of the standard being defined by the implementation.

I wouldn't want to cause a bunch of work to deal with this, fairly fringe case (we have a lot of skipped files in our repo and it's helpful to group them on different lines for readability). So feel free to close this WONTFIX unless you want to pursue it - thanks!

**Comment by DimitriPapadopoulos** [comment]
The fix is simple (#3400), and is very much related to the fix for #2727 (#2767).

I just need to find the time to write tests before #2767 and #3400 can be merged.
